recipe/recipe.yaml

```
@@ -1,51 +1,52 @@
-{% set name = "awkward" %}
-{% set version = "2.8.3" %}
-{% set python_min = "3.9" %}
+schema_version: 1
+
+context:
+  name: awkward
+  version: 2.8.3
+  python_min: 3.9
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: ${{ name|lower }}
+  version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/awkward-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/awkward-${{ version }}.tar.gz
   sha256: 31db6f8d923a844791d6bfc233397fd4b34c72ade47852727b75278f1351f904
 
 build:
+  number: 1
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: ${{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python {{ python_min }}
+    - python 3.9.*
```

```suggestion
    - python ${{ python_min }}.*
```

Bug from https://github.com/conda/conda-recipe-manager/issues/307